### PR TITLE
feat(metadata-sidebar): adjust extractSuggestions to use metadata_template

### PR DESCRIPTION
**Pull request by wpiesiak**
Adjust extractSuggestions function to use metadata_template instead of fields

### CODE REVIEW

**Review by jankowiakdawid — DISMISSED** [review]

File: src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts [hunk 1]
```diff
@@ -222,8 +222,12 @@ function useSidebarMetadataFetcher(
             if (isEmpty(answer)) {
                 const error = new Error('No suggestions found.');
                 onError(error, ERROR_CODE_EMPTY_METADATA_SUGGESTIONS, { showNotification: true });
+                return [];
             }
 
+            const fields =
+                templateInstances.find(template => template.templateKey === templateKey && template.scope === scope)
```
Comment (bfoxx1906) [inline src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts:229]: nit: this line is very long and complex. I would recommend extracting out the templateInstances filter into it's own value e.g.
` 
       const template =  templateInstances.find(template => template.templateKey === templateKey && template.scope === scope);
const fields = template?.fields || [] 
`

Comment (wpiesiak) [reply src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts]: done 👍

**Review by bfoxx1906 — DISMISSED** [review]
lgtm

File: package.json [hunk 2]
```diff
@@ -128,7 +128,7 @@
         "@box/cldr-data": "^34.2.0",
         "@box/frontend": "^10.0.0",
         "@box/languages": "^1.0.0",
-        "@box/metadata-editor": "^0.73.0",
+        "@box/metadata-editor": "0.78.3",
```
Comment (greg-in-a-box) [inline package.json:131]: can we update the peer dep too?

**Review by greg-in-a-box — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (wpiesiak) [reply package.json:131]: Will fix in the following PR

Comment (wpiesiak) [reply package.json:131]: done

**Review by kajarosz — APPROVED** [review]

**Review by greg-in-a-box — APPROVED** [review]
i dont think it matters but the devdep is fixed while the peer dep is ^, approving anyways since the next pr should have ^ on both

**Review by bfoxx1906 — APPROVED** [review]
LGTM

**Pull request merged**